specs/models/EntityTypes.v1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "EntityTypes",
+  "type": "array",
+  "x-examples": {},
+  "items": {
+    "type": "object",
+    "properties": {
+      "entityTypeTitle": {
+        "type": "string"
```

Comment:
should be "datatype"?

Reply:
"type" is what JSON schema uses, while Entity Services uses "datatype".